src/dot_claude/commands/claude/new/command.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Interactive builder for creating new Claude slash commands
+argument-hint: [optional initial context]
+---
+
+You are an expert at creating Claude slash commands. Your goal is to help the user create a new slash command file in this repository.
+
+## Process
+
+1.  **Determine the Command Name and Path**:
+    *   If the user did not provide a name, ask for it (e.g., "What should the slash command be? Example: `/git:status`").
+    *   Convert the command name to a file path inside `src/dot_claude/commands/`.
+    *   **Rules**:
+        *   Colons (`:`) or slashes (`/`) map to directory separators.
+        *   The file extension must be `.md`.
+        *   Example: `/git:status` -> `src/dot_claude/commands/git/status.md`.
+        *   Example: `/utils/date` -> `src/dot_claude/commands/utils/date.md`.
+
+2.  **Gather Command Details**:
+    *   **Description**: Ask for a short description (required for the frontmatter).
+    *   **Purpose**: Ask what the command should do. What instructions should it give to the agent?
+    *   **Arguments**: (Optional) Ask if the command requires any arguments (for the `argument-hint` frontmatter).
+
+3.  **Draft the File Content**:
+    *   Construct the file content with valid YAML frontmatter and the user's instructions.
+    *   Use ventilated prose (one sentence per line) for the instructions in the body of the command, as per repository conventions.
+
+    **Template**:
+    ```markdown
+    ---
+    description: {description}
+    argument-hint: {argument_hint}
+    ---
+
+    {instructions}
+    ```
+
+4.  **Review and Write**:
```

Comment:
We should also indicate how exclamation and at sign @ work within hooks to embed files and read them and also execute commands as part of the command